code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/Consent.java

```diff
@@ -15,18 +15,24 @@
 import com.adobe.marketing.mobile.AdobeCallbackWithError;
 import com.adobe.marketing.mobile.AdobeError;
 import com.adobe.marketing.mobile.Event;
-import com.adobe.marketing.mobile.ExtensionError;
-import com.adobe.marketing.mobile.ExtensionErrorCallback;
-import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.EventSource;
+import com.adobe.marketing.mobile.EventType;
+import com.adobe.marketing.mobile.Extension;
 import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.services.Log;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 public class Consent {
 
+	private static final String CLASS_NAME = "Consent";
+	public static final Class<? extends Extension> EXTENSION = ConsentExtension.class;
```

Comment:
nit:  suggest re-ordering to have `public static` members before  `private static`